<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1095</width>
    <height>675</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QStackedWidget" name="stackedwidget_main">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>0</y>
     <width>911</width>
     <height>671</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>4</number>
   </property>
   <widget class="QWidget" name="win_select">
    <widget class="QGroupBox" name="groupbox_0">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>891</width>
       <height>651</height>
      </rect>
     </property>
     <property name="title">
      <string>选择窗口</string>
     </property>
     <widget class="QLabel" name="inf_select">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>310</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;未选择窗口&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="data_get">
    <widget class="QGroupBox" name="groupbox_1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>891</width>
       <height>651</height>
      </rect>
     </property>
     <property name="title">
      <string>窗口一</string>
     </property>
     <widget class="QLabel" name="label_video">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>601</width>
        <height>601</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;监控&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushbutton_video_close">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>40</y>
        <width>61</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>关闭</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupbox_1_inf">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>30</y>
        <width>201</width>
        <height>601</height>
       </rect>
      </property>
      <property name="title">
       <string>详细信息</string>
      </property>
      <widget class="QLabel" name="label_temp">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>30</y>
         <width>101</width>
         <height>41</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::WinPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;温度&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_temp_data">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>80</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_water">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>120</y>
         <width>101</width>
         <height>41</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::WinPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;湿度&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_water_data">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>170</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_door">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>210</y>
         <width>101</width>
         <height>41</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::WinPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;门的开关&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_door_data">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>270</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_foodtime">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>320</y>
         <width>101</width>
         <height>41</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::WinPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;上次投喂时间&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_foodtime_data">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>370</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_watertime">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>420</y>
         <width>101</width>
         <height>41</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::WinPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;上次喂水时间&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_watertime_data">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>470</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_virtime">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>510</y>
         <width>101</width>
         <height>41</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::WinPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;上次消毒时间&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_virtime_data">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>560</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="data_history">
    <widget class="QGroupBox" name="groupbox_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>891</width>
       <height>651</height>
      </rect>
     </property>
     <property name="title">
      <string>历史数据</string>
     </property>
     <widget class="QTextBrowser" name="textbrowser">
      <property name="geometry">
       <rect>
        <x>25</x>
        <y>31</y>
        <width>841</width>
        <height>601</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="video_trans">
    <widget class="QGroupBox" name="groupbox_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>891</width>
       <height>651</height>
      </rect>
     </property>
     <property name="title">
      <string>视频与文字传输(服务器)</string>
     </property>
     <widget class="QLabel" name="label_video_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>611</width>
        <height>601</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;视频传输&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupbox_3_inf">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>20</y>
        <width>231</width>
        <height>141</height>
       </rect>
      </property>
      <property name="title">
       <string>详细信息</string>
      </property>
      <widget class="QLabel" name="label_ip">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::WinPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;本机IP地址&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_udp_port">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::WinPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;UDP监听端口&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_tcp_port">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>100</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::WinPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;TCP监听端口&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_ip_inf">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>20</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_udp_port_inf">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>60</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_tcp_port_inf">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>100</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="push_listen_begin">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>170</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>开始监听</string>
      </property>
     </widget>
     <widget class="QPushButton" name="push_image_cat">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>210</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>截屏</string>
      </property>
     </widget>
     <widget class="QPushButton" name="push_video_cat">
      <property name="geometry">
       <rect>
        <x>770</x>
        <y>210</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>录屏</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineedit_path_change">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>250</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="push_path_change">
      <property name="geometry">
       <rect>
        <x>770</x>
        <y>250</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>更改保存路径</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radio_jpg_select">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>290</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>JPG</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="radio_png_select">
      <property name="geometry">
       <rect>
        <x>770</x>
        <y>290</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>PNG</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="push_log_clear">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>610</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>清空</string>
      </property>
     </widget>
     <widget class="QPushButton" name="push_msg_send">
      <property name="geometry">
       <rect>
        <x>770</x>
        <y>610</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>发送</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="textBrowser_2">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>320</y>
        <width>231</width>
        <height>241</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="text_edit_msg">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>570</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="database_opr">
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>891</width>
       <height>651</height>
      </rect>
     </property>
     <property name="title">
      <string>数据库操作</string>
     </property>
     <widget class="QLabel" name="label_id">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>70</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:600;&quot;&gt;学号:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_name">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>120</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:600;&quot;&gt;姓名:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_bir">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>170</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:600;&quot;&gt;生日:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineedit_id">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>60</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineedit_name">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>110</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineedit_bir">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>160</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="push_select">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>220</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>查询</string>
      </property>
     </widget>
     <widget class="QPushButton" name="push_insert">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>220</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>插入</string>
      </property>
     </widget>
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>60</y>
        <width>481</width>
        <height>141</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="push_insert_view">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>220</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>查询数据库</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupbox_main">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>161</width>
     <height>651</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="title">
    <string>功能选择</string>
   </property>
   <widget class="QPushButton" name="push_video">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>视频传输</string>
    </property>
   </widget>
   <widget class="QComboBox" name="combobox_main">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
    <property name="currentText">
     <string>选择窗口</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <item>
     <property name="text">
      <string>选择窗口</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>窗口1</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="push_dataget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>获取数据</string>
    </property>
   </widget>
   <widget class="QPushButton" name="push_history">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>历史数据</string>
    </property>
   </widget>
   <widget class="QPushButton" name="push_database">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>数据库</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
